test/cookbooks/python3-test/recipes/default.rb

```diff
@@ -4,16 +4,26 @@
 #
 # Copyright:: 2022, The Authors, All Rights Reserved.
 
-python_install node['python3']['name']
+node['python3_test']['pythons'].each_with_index do |python, idx|
```

Comment:
OPT: instead of `.each_with_index` you could have use a Hash, using the key for the virtualenv name